package.json

```diff
@@ -46,9 +49,10 @@
 	"homepage": "https://github.com/progfay/scrapbox-parser#readme",
 	"devDependencies": {
 		"@biomejs/biome": "1.9.4",
+		"@types/node": "22.15.2",
```

Comment:
Requirement to install `@types/node` package caused by uninstalling `vite` package.